schema/project-level/project-schema.json

```diff
@@ -1436,6 +1460,141 @@
         "id"
       ]
     },
+    "Metric": {
+      "type": "object",
+      "title": "Metric",
+      "description": "Metrics are used to set out forecast and actual metrics targets for a project, for example planned and actual physical and financial progress over time.",
```

Comment:
Need to update to match change in borrow-schema.py (I think running the script again might delete uniqueItems again - maybe easier to just hand-edit this time)

Reply:
Good spot. The output of `borrow-schema.py` had diverged from `project-schema.json` in a couple of other places too, so I've updated the script and run it in https://github.com/open-contracting/infrastructure/pull/282/commits/7fe856f13121ff731d90dd8df57aa0168d7889c0.